R/drive_trash.R

```diff
@@ -0,0 +1,100 @@
+#' Trash file on Drive.
+#' @template file
+#' @template verbose
+#'
+#' @return Logical, indicating whether the trashing/untrashing succeeded.
+#' @export
+#' @examples
+#' \dontrun{
+#' ## Create a file to trash.
+#' file <- drive_upload(system.file("DESCRIPTION"), "DESC")
+#' ## Put a single file in the trash.
+#' drive_trash("DESC")
+#' drive_untrash("DESC")
+#'
+#' ## Put multiple files in trash.
+#' file_a <- drive_upload(system.file("DESCRIPTION"), "file_a")
+#' file_b <- drive_upload(system.file("DESCRIPTION"), "file_b")
+#' drive_trash(c("file_a", "file_b"))
+#' drive_untrash(c("file_a", "file_b"))
+#' }
+drive_trash <- function(file = NULL, verbose = TRUE) {
+  trash_file <- as_dribble(file)
+  if (!some_files(trash_file) && verbose) {
```

Comment:
`no_file()` is nicer than `!some_files()`

Should this return here? And invisibly return `logical(0)`?